.github/workflows/dev-env-api-test.yaml

```diff
@@ -0,0 +1,36 @@
+name: Dev env internal API test
```

Comment:
This is just hear to demonstrate the PR works, and how to access the API key programatically.